src/app/auth/components/sign-up/sign-up.component.ts

```diff
@@ -62,8 +66,23 @@ export class SignUpComponent implements OnInit {
 
   onSignUpClick(): void {
     this.userService.userRegistration(this.form.value)
-        .pipe(first())
-        .subscribe(res => console.log(res));
+      .pipe(first())
+      .subscribe(
```

Comment:
.subscribe(
        (res: Partial<NotificationMessage>) => this.handleUserRegistrationSuccessResponse(res),
        (err) => this.handleUserRegistrationErrorResponse(res)
      );